### PR TITLE
Pseudo-Irradiance Testing

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Add support for output from pseudo-irradiance forecasting model, and test the result on a few unseen-to-irradiance model sites. The PV site model is trained on 2020 and tested on 2021 for the available sites.

Fixes #

## How Has This Been Tested?

Running the code and experiment

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by jacobbieker** [comment]
The results might be slightly skewed, as there might be some issues with the data prep that has been introduced. After checking out the main branch and running PV-only training there, that error is 0.313 on the validation set of 2021 vs 0.475 here. There are some difference, like the sites that the models were trained on, and the data is more random (any start time vs the ones that are in the irradiance training set) but that is still a large difference. The shapes of the outputs are also similar, but still different enough that something seems to be wrong.

**Comment by jacobbieker** [comment]
The error is actually smaller if using only the same IDs: 0.250 for the main-branch PV-only forecast. So the difference is even larger, which is odd
